data/data/alibabacloud/cluster/vpc/vpc.tf

```diff
@@ -19,12 +27,12 @@ resource "alicloud_vpc" "vpc" {
   )
 }
 
-resource "alicloud_vswitch" "vswitchs" {
-  count = length(var.zone_ids)
+resource "alicloud_vswitch" "vswitches" {
+  count = length(var.vswitch_ids) == 0 ? length(var.zone_ids) : 0
```

Comment:
This count is not correct when the installer is creating the vSwitches. Using `length(var.zone_ids)` assumes that the zones in `var.zone_ids` are unique. If the master replica count is greater than the number of availability zones used, then this will create multiple vSwitches in some of the zones, and those extras will be omitted from the mapping from zone ID to vSwitch ID.

Reply:
I can solve this by `length(toset(var.zone_ids))`, but I think this may not be the most suitable solution. I should ensure that the zones in var_zone_ids is unique,  at the same time, the length of ali_zone_ids need not be equal master_count. WDYT?

Reply:
Yes, `var.zone_ids` should be a distinct list of availability zones. It should also include zones for the workers, which appears to be missing currently.

Reply:
Yes, thank you for your reminder, the worker zones is missing currently, I will create a new PR to update.

Reply:
> Yes, var.zone_ids should be a distinct list of availability zones. It should also include zones for the workers, which appears to be missing currently.

@staebler #5438